tgui/packages/tgui/styles/components/Divider.scss

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2020 Aleksej Komarov
+ * SPDX-License-Identifier: MIT
+ */
+
+ @use '../base.scss';
+
+$color: rgba(255, 255, 255, 0.1) !default;
+$thickness: base.em(2px) !default;
+$spacing: 0.5em;
+
+.Divider--horizontal {
+  margin: $spacing 0;
+
+  &:not(.Divider--hidden) {
```

Comment:
Setting the border when it's "not hidden" seems like a weird order to me, I might be tweaking this to set the border to what it needs to be, then setting the width to `0` if it's hidden.

Reply:
Good call.